.circleci/config.yml

```diff
@@ -1,59 +1,74 @@
-version: 2.1
+version: 2.1  # Version of CircleCI config format
 
-orbs:
-  codecov: codecov/codecov@1.1.0
+orbs:  # CircleCI packages ("orbs") we depend on
```

Comment:
Would be useful to clarify here what an "orb" is, and what level the package exists on

Reply:
What I meant was: packages generally exist at a particular level of a system (you have Python packages, and separately binary-installed packages). Maybe, by analogy, if I was reading this, I might wonder if an Orb has some comparability to a virtualenv or a Docker image, in that it sets up machine/environment dependencies, or if the packages it contains are purely relevant to the relationship with CircleCI itself

Reply:
Oh I see, I hadn't even considered that interpretation, I've rephrased -- does this make sense now?